### ISSUE TITLE
Accessibility: Pages should have titles to describe purpose

### ISSUE THREAD

**Issue by aduth** [GSA/centers-of-excellence#357]

**Comment by iamjolly** [comment]
@daniellegallant - This one is ready for your efforts within the markdown page files. In the top part of each file, there will be a front-matter section denoted by dashes at the beginning and end of the front-matter section. You should now be able to add:

`title: Something Short and Descriptive for Title of the Page`

for any pages on the site. You can also edit existing page titles. This will then allow every page to have a unique, correct page title which will help with accessibility and SEO.
